### PR TITLE
Added test for asyncdr mongo crs

### PR DESCRIPTION
**Pull request by gejain**
**What this PR does / why we need it**:
Added test for asyncdr mongo crs
**Which issue(s) this PR fixes** (optional)
Closes #PTX 18688

**Special notes for your reviewer**:
Results - https://aetos.pwx.purestorage.com/resultSet/testSetID/288643/testCaseID/1595285

### CODE REVIEW

File: tests/testTriggers.go [hunk 1]
```diff
@@ -7195,6 +7197,159 @@ func TriggerKafkaAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *Event
...
+		if err != nil {
+			UpdateOutcome(event, fmt.Errorf("Failed to set source config, err: %v", err))
+			return
+		}
+	})
+	stepLog = fmt.Sprintf("Deploy applications with %v chaos level", chaosLevel)
+	Step(stepLog, func() {
+		log.InfoD(stepLog)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			//taskName := fmt.Sprintf("%s-%d", taskNamePrefix, i)
```
Comment (lsrinivas-pure) [inline tests/testTriggers.go:7250]: remove if not required

File: tests/testTriggers.go [hunk 2]
```diff
@@ -7195,6 +7197,159 @@ func TriggerKafkaAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *Event
...
+				log.InfoD("Migration Started")
+				mig_name := migrationKey + "mongo-" + fmt.Sprintf("%d", i) + time.Now().Format("15h03m05s")
+				mig, err := asyncdr.CreateMigration(mig_name, ns.Name, asyncdr.DefaultClusterPairName, ns.Name, &includeVolumesFlag, &includeResourcesFlag, &startApplicationsFlag)
+				if err != nil {
+					UpdateOutcome(event, fmt.Errorf("Failed to create migration: %v", err))
+					return
+				}
+				migrationList = append(migrationList, mig)
+				err = asyncdr.WaitForMigration(migrationList)
+				if err == nil {
```
Comment (lsrinivas-pure) [inline tests/testTriggers.go:7311]: instead of err == nil check do err !=nil check and return if err
so that we can avoid using else

Comment (gejain) [reply tests/testTriggers.go]: Done

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**